dev/MRTCore/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+Copyright (c) Microsoft Corporation.
```

Comment:
This should already be covered by the license file in the root of the repo.

Reply:
before we figure out how to handle the child projects, I'd rather leave it self contained. Also this file is referenced by nuspec.

---
In reply to: [488400196](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488400196) [](ancestors = 488400196)

Reply:
Except that you're not self-contained, you're now part of Project Reunion.  In a future change, please remove this file and point your nuspec at the one in the root.

Reply:
yes. this will be handled when we enable CI build.

---
In reply to: [488958359](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488958359) [](ancestors = 488958359)